src/App.js

```diff
@@ -1,15 +1,32 @@
 import "./App.css";
 import React, { useState } from "react";
-import PlayerTurn from "./components/PlayerTurn";
-import EndGameDeclaration from "./components/EndGameDeclaration";
-import NewGame from "./components/NewGame";
-import Board from "./components/Board";
+import {
+  PlayerTurn,
+  EndGameDeclaration,
+  NewGame,
+  Board,
+  winLogic,
+} from "./exporter";
```

Comment:
Nice. Some people just reference path to "components/index" and index is that export file that is a manifest of all components. Good find.